.github/workflows/build.yml

```
@@ -68,13 +68,20 @@ jobs:
         if: runner.os == 'macOS'
         run: |-
           brew install xerces-c qt5 qwt-qt5
+      - name: setup for run-vcpkg
+        if: runner.os == 'Windows'
+        shell: cmd
+        run: |
+          echo { "name": "opendds", "version-string": "github-actions", "dependencies": [ "qt5-base", "openssl", "xerces-c", "qwt" ] } > vcpkg.json
```

```suggestion
          echo '{ "name": "opendds", "version-string": "github-actions", "dependencies": [ "qt5-base", "openssl", "xerces-c", "qwt" ] }' > vcpkg.json
```
Think you need the single quotes to preserve the double quotes.